CONTRIBUTORS.md

```diff
@@ -84,4 +85,5 @@ Community
 [@saket2403]: https://github.com/saket2403
 [@DMcginn]: https://github.com/DMcginn
 [@ashishkcerner]: https://github.com/ashishkcerner
-[@razvanluca] https://github.com/razvanluca
\ No newline at end of file
+[@razvanluca] https://github.com/razvanluca
```

Comment:
Can you add ":" to the link above?

Reply:
Updated here - https://github.com/cerner/terra-clinical/pull/858/commits/56ee63b0c7ee1c47753b8b681f57337c9009eb41#diff-c0f86987c556ec52d97b9acf0f35bb2ad0521f65c3113e1b15362ca76502eed2R88